pkg/cmd/server/api/v1/types_test.go

```diff
@@ -351,7 +353,8 @@ func TestMasterConfig(t *testing.T) {
 		AssetConfig: &internal.AssetConfig{
 			Extensions: []internal.AssetExtensionsConfig{{}},
 		},
-		DNSConfig: &internal.DNSConfig{},
+		DNSConfig:             &internal.DNSConfig{},
+		AdmissionPluginConfig: map[string]string{"pluginName": "configFilePath"}, // Must be specified explicitly becase it's omitempty
```

Comment:
@liggitt I just realized what you were saying is not what I understood :) so yeah I agree with @deads2k... the separate config seems better, since most plugins don't require a config.

What I thought you were saying was that we inline the config for each plugin in the master config file